libsolidity/formal/BMC.cpp

```diff
@@ -49,6 +49,8 @@ void BMC::analyze(SourceUnit const& _source, shared_ptr<Scanner> const& _scanner
 
 	m_scanner = _scanner;
 
+	m_variableUsage.setFunctionInlining(true);
```

Comment:
Where is it reset?

Reply:
Each engine should set it before its execution.
In this case, when added, CHC will set it to false in a similar way.

Reply:
Why not make it part of the constructor then?